hushline/__init__.py

```diff
@@ -3,28 +3,22 @@
 from datetime import timedelta
 from typing import Any
 
-from dotenv import load_dotenv
 from flask import Flask, flash, redirect, render_template, session, url_for
 from flask_limiter import RateLimitExceeded
-from flask_migrate import Migrate, upgrade
+from flask_migrate import Migrate
 from werkzeug.wrappers.response import Response
 
 from . import admin, routes, settings
 from .db import db
 from .limiter import limiter
 from .model import User
 
-load_dotenv("/etc/hushline/hushline.conf")
```

Comment:
is this no longer needed because the environment variables are actually provided in the environment?

Reply:
Yes. Also, it wasn't actually even being used before, as there was no `/etc/hushline/hushline.conf` to load.